habitat-lab/habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -143,23 +143,28 @@ class HabitatSimDepthSensor(DepthSensor, HabitatSimSensor):
     max_depth_value: float
 
     def __init__(self, config: DictConfig) -> None:
+        self.min_depth_value = config.min_depth
+        self.max_depth_value = config.max_depth
+        self.normalize_depth = config.normalize_depth
         if config.normalize_depth:
```

Comment:
Wont matter a lot but you can change to:
 ```suggestion
        if self.normalize_depth:
```